chore: call out that that the vscode extension is for Node.js (#32882)
microsoft/playwright: docs/src/getting-started-vscode-js.md

## Installation

> To use Playwright for Node.js from VS Code, install the [VS Code extension from the marketplace](https://marketplace.visualstudio.com/items?itemName=ms-playwright.playwright) or from the extensions tab in VS Code.

Author: ```suggestion
Playwright has a VS Code extension which is available when testing with Node.js. Install [it from the VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=ms-playwright.playwright) or from the extensions tab in VS Code.
```

as per debbie's suggestion
